### ISSUE TITLE
how apply keras tuner with fine tuning?

### ISSUE THREAD

**Issue by inginfor** [keras-team/keras-tuner#1043]
I am fine tuning starting from mobilnet, so in a first phase I train only the classifier, then the upper layers and in a second phase I unfreeze some layers of the pre-trained network and continue training. I need to optimize the hyperparameters with keras tuner the only problem is that I have not found any example applied to fine tuning. So, regarding the hyperparameter optimization for the two phases, the first one of feature extraction and the second one of fine tuning by unfreezing the layers, how should the hyperparameter optimization be performed if you are fine tuning? For example, in the first phase you need a certain initial number of epochs and a higher learning rate, while in the second phase another number of epochs and a lower learning rate. How to automate the search for the best hyperparameters in fine tuning with keras tuner?
